job_templates/ci_launcher_job.xml.em

```diff
@@ -2,7 +2,7 @@
 <project>
   <actions/>
   <description>
-    Note: The Windows jobs ignore the middleware selection flags.
+    Launches a build of the CI job for each supported platform with the same set of parameters.
```

Comment:
`aarch64` ignore Connext atm. Not sure if it makes sense to mention it here though...